internal/services/device_data.go

```diff
@@ -9,22 +9,35 @@ import (
 	"time"
 
 	"github.com/DIMO-Network/rewards-api/internal/config"
+	"github.com/DIMO-Network/rewards-api/internal/services/ch"
 	"github.com/aquasecurity/esquery"
 	"github.com/elastic/go-elasticsearch/v7"
 )
 
+const integPrefix = "dimo/integration/"
+
 type DeviceDataClient interface {
-	GetLastActivity(userDeviceID string) (lastActivity time.Time, seen bool, err error)
-	DescribeActiveDevices(start, end time.Time) ([]*DeviceData, error)
+	DescribeActiveDevices(start, end time.Time) ([]*ch.Devices, error)
 	GetIntegrations(userDeviceID string, start, end time.Time) (ints []string, err error)
 }
```

Comment:
I think we can just delete this interface since it isn't being used by anyone then make the struct public.

Reply:
It's not a good use but it's a use

https://github.com/DIMO-Network/rewards-api/blob/7f745bd5be85034df6d13318a743d143f527adfd/internal/controllers/rewards.go#L30